README.md

```diff
@@ -101,23 +101,32 @@ git clone git@github.com:roboflow-ai/roboflow-python.git && cd roboflow-python
 # Be sure to update the values with your account's information
 cp .env-example .env
 
-# Build and run a new docker container image
-docker run -it -v $(pwd):/roboflow --name roboflow python:3.8.14-slim /bin/bash
+# Build our development image
+docker build -t roboflow-python -f Dockerfile.dev .
 
-# Install Dependencies
-cd roboflow/ && pip install -e ".[dev]"
+# Run container and map current folder in it
+docker run --rm -it \
+  -e .env
```

Comment:
after the mocking PR is merged, i think that we will have no need for `.env` or `.env-example`. those values were only used in tests and are no longer necessary 😄

Reply:
nice catch! Thanks. Updated :)